cookbooks/supervisor/recipes/default.rb

```diff
@@ -27,18 +25,8 @@
   end
 end
 
-python_package 'meld3' do
-  action :upgrade
-  version '1.0.0'
-end
-
-python_package 'supervisor' do
-  action :upgrade
-  version '3.3.1'
-end
-
-python_package 'setuptools' do
-    action :upgrade
+execute 'pip install supervisor' do
+   command 'pip install supervisor --index=https://pypi.python.org/simple/'
```

Comment:
Perhaps a specific version might be helpful to lock here? e.g. `pip install supervisor==3.3.1 ...` to prevent surprises on supervisor upgrades